### DISCUSSION TITLE
Upgrade from personal to Enterprise

### DISCUSSION THREAD

**Discussion by moondustpublishing** [community/community#38888] [Enterprise]
### Select Topic Area

Question

### Body

If I upgrade from personal account to Enterprise do I keep the same account and/or just add on features or do I need to delete everything and start over???

**Comment by jfullstackdev** [comment]
this is maybe what you are asking for

![image](https://user-images.githubusercontent.com/47092464/201454643-6f25eb83-c55a-4ab3-a90a-cfd4effbfc2f.png)

https://docs.github.com/en/enterprise-cloud@latest/admin/overview/about-enterprise-accounts
